nav2_system_tests/src/system/CMakeLists.txt

```diff
@@ -11,6 +11,6 @@ ament_add_test(test_bt_navigator
     TEST_WORLD=${PROJECT_SOURCE_DIR}/worlds/turtlebot3_ros2_demo.world
     TEST_PARAMS=${PROJECT_SOURCE_DIR}/params/nav2_params.yaml
     GAZEBO_MODEL_PATH=${PROJECT_SOURCE_DIR}/models
-    BT_NAVIGATOR_XML=navigate_w_recovery_retry.xml
+    BT_NAVIGATOR_XML=navigate_w_replanning_and_recovery.xml
```

Comment:
Our default tree would be `navigate_w_replanning.xml`

Reply:
Why no recovery? We have been using it on master branch